src/MethodProxies-Tests/MpConcurrencyHandler1.class.st

```diff
@@ -0,0 +1,10 @@
+Class {
+	#name : #MpConcurrencyHandler1,
```

Comment:
Let's choose better names. This handler is there to wait until the thread execution starts, right? let's choose a name in that direction :)